velox/exec/tests/NestedLoopJoinTest.cpp

```diff
@@ -15,15 +15,54 @@
  */
 #include "velox/exec/tests/utils/HiveConnectorTestBase.h"
 #include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/exec/tests/utils/VectorTestUtil.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
 
 using namespace facebook::velox;
 using namespace facebook::velox::exec;
 using namespace facebook::velox::exec::test;
 
 class NestedLoopJoinTest : public HiveConnectorTestBase {
  protected:
+  RowTypePtr leftType_;
+  RowTypePtr rightType_;
+  std::vector<std::string> comparisons_;
+  std::vector<core::JoinType> joinTypes_;
+  std::string joinConditionStr_;
+  std::string queryStr_;
+
   void SetUp() override {
     HiveConnectorTestBase::SetUp();
+    leftType_ = ROW({{"t0", BIGINT()}});
+    rightType_ = ROW({{"u0", BIGINT()}});
+    comparisons_ = {"=", "<", "<=", "<>"};
+    joinTypes_ = {
+        core::JoinType::kInner,
+        core::JoinType::kLeft,
+        core::JoinType::kRight,
+        core::JoinType::kFull};
+    joinConditionStr_ = "t0 {} u0";
+    queryStr_ = "SELECT t0, u0 FROM t {} JOIN u ON t.t0 {} u.u0";
+  }
+
+  void setLeftType(RowTypePtr leftType) {
```

Comment:
nit: can we use setProbeType for setLeftType and setBuildType for setRightType to be consistent with hash join test. Thanks!

Reply:
I've changed all the left/right naming to probe/build to be consistent with HashJoinTest, thanks!